addons/mail/models/mail_message.py

```diff
@@ -1070,18 +1077,15 @@ def _to_store(self, store: Store, fields, *, format_reply=True, msg_vals=False,
             # model, res_id, record_name need to be kept for mobile app as iOS app cannot be updated
```

Comment:
comment can probably be removed now, as it is present on each individual line on _to_store_defaults